### PR TITLE
Messages API

### PR DESCRIPTION
**Pull request by mrprfrm**
Messages CRUD API, Messages API tests, mongodb implementation.

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,44 +1,126 @@
...
+async def get_db():
+    client = AsyncIOMotorClient(settings.db_url, uuidRepresentation="standard")
+    yield client[settings.db_name]
+    client.close()
+
+
 app = FastAPI()
 
+app.add_middleware(
+    CORSMiddleware,
```
Comment (kuzminivan) [inline app/main.py:31]: @mrprfrm what is this "CORSMiddleware"?

File: app/main.py [hunk 2]
```diff
@@ -1,44 +1,126 @@
...
 
+async def get_db():
+    client = AsyncIOMotorClient(settings.db_url, uuidRepresentation="standard")
+    yield client[settings.db_name]
+    client.close()
+
+
 app = FastAPI()
 
+app.add_middleware(
```
Comment (kuzminivan) [inline app/main.py:30]: @mrprfrm  what is this "CORSMiddleware"?

**Review by kuzminivan — COMMENTED** [review]
Please, take a look on my comment

File: app/main.py (hunk 1 above)
Comment (mrprfrm) [reply app/main.py:31]: CORS is Cross Origin Resource Sharing, we need it to make possible using current API with browser clients.

File: app/main.py [hunk 3]
```diff
@@ -1,44 +1,126 @@
...
+    CORSMiddleware,
+    allow_origins=settings.origins,
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+
+class MessagesCollection(pydantic.BaseModel):
```
Comment (kuzminivan) [inline app/main.py:39]: why do you use this particullar class?

**Review by kuzminivan — APPROVED** [review]
Ok, everything is good

**Pull request merged**